src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignments/test_lmsassignments.py

```diff
@@ -98,11 +98,48 @@ def it_should_run_successfully(test_db_config: ServerConfig, source_system: str)
 
         # assert
         with MSSqlConnection(test_db_config).pyodbc_conn() as connection:
-            LMSAssignment = query(connection, "SELECT * from [lmsx].[Assignment]")
-            assert len(LMSAssignment) == 1
-            assert (
-                int(LMSAssignment[0]["AssignmentIdentifier"]) == 1
-            )  # It is using the identity field from lms.Assignment
+            result = query(connection, "SELECT * from [lmsx].[Assignment]")
+
+        assert len(result) == 1, "There should be one result."
+
+        LMSAssignment = result[0]
+
+        assert (
+            int(LMSAssignment["AssignmentIdentifier"]) == 1
+        ), "It should map the assignment identifier"
+
+        # We know the id of the descriptors based in the order how they are inserted.
+        assert (
+            int(LMSAssignment["LMSSourceSystemDescriptorId"]) == 2
+        ), "It should map the SourceSystem descriptor"
+
+        assert (
+            int(LMSAssignment["AssignmentCategoryDescriptorId"]) == 1
+        ), "It should map the assignment category descriptor"
+
+        assert (
+            LMSAssignment["SectionIdentifier"] == SIS_SECTION_ID
+        ), "It should map the section identifier"
+
+        assert (
+            LMSAssignment["LocalCourseCode"] == 'Local course code test'
+        ), "It should map the local course code"
+
+        assert (
+            LMSAssignment["SessionName"] == 'session name test'
+        ), "It should map the SessionName"
+
+        assert (
+            int(LMSAssignment["SchoolYear"]) == 1
+        ), "It should map the SchoolYear"
+
+        assert (
+            int(LMSAssignment["SchoolId"]) == 1
+        ), "It should map the SchoolId"
+
+        assert (
+            LMSAssignment["Namespace"] is not None
```

Comment:
In fact, we do know the source system - it was defined in a function call above.